### PR TITLE
Bug 1252816 - Load the statusbar module only when making a phone build

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=de77342d6aa0a8d3a8c3116352a0d715ead7cbbb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=90b4298e1f16b8f3a83373fba46f236de45b9a39

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5408103400300679c5d2d6fe63d23f330178ac59

File: build/csslint/xfail.list [hunk 1]
```diff
@@ -108,6 +108,7 @@ apps/system/style/modal_dialog/modal_dialog.css
 apps/system/style/notifications/notifications.css
 apps/system/style/software_button_manager/software_button_manager.css
 apps/system/style/sound_manager/sound_manager.css
+apps/system/style/statusbar/statusbar.css
```
Comment (mikehenrty) [inline build/csslint/xfail.list]: Why do we need to add this to the xfail list when you have the `/*csslint empty-rules: false*/` above?

File: apps/system/style/definitions.css [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /* See bug 1053721 */
 :root {
   --software-home-button-height: 5rem;
-  --statusbar-height: 3rem;
+  --statusbar-height: 0rem;
```
Comment (mikehenrty) [inline apps/system/style/definitions.css]: nice

File: apps/system/js/core.js [hunk 3]
```diff
@@ -110,11 +109,12 @@
         'MultiScreenController',
         'PinsManager',
         //IFDEF_FIREFOX_SYNC
-        'SyncManager'
+        'SyncManager',
         //ENDIF_FIREFOX_SYNC
         // XXX: We should move CarrierInfoNotifier into mobileConnectionCore,
         // but integration tests running on desktop without mobileConnection
         // is testing this.
+        'CustomModules'
```
Comment (mikehenrty) [inline apps/system/js/core.js]: I guess this means statusbar only loads when idle. Are we sure that is what we want for the smartphone boot?

Comment (albertopq) [inline apps/system/js/core.js]: That was the same it was doing, wasn't it? https://github.com/mozilla-b2g/gaia/pull/34118/files#diff-135794ad5f1433b712b86b905dbfab95L70

File: build/csslint/xfail.list (hunk 1 above)
Comment (albertopq) [inline build/csslint/xfail.list]: It still fails returning a RBRACE error. Couldn't find another solution... May be we can remove the ignore rule in the file, though

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f7d667b9c9929a49e6db0daf1be7e96c25621014

**Pull request closed without merging**